Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN yarn run build
 #
 FROM node:16-alpine as production
 USER node
+COPY --chown=node:node --from=base /app/package.json ./package.json
```

Comment:
This is necessary as we need to access `package.json` inside the container to gather the data. I tested it by building the container in my local setup.